eos_data_distribution/ndn/base.py

```diff
@@ -139,10 +139,12 @@ def generateKeys(self, name=None):
     def sign(self, data):
         return self._keyChain.sign(data, self._certificateName)
 
-    def expressInterest(self, interest=None, *args, **kwargs):
+    def expressInterest(self, interest=None, name=None, *args, **kwargs):
         if interest is None:
             interest = Interest(self.name)
```

Comment:
Shouldn’t this be changed to use `interest = Interest(name)` after `name` has been set up?